README.md

```diff
@@ -0,0 +1,16 @@
+# Crater Atomfeed
+
```

Comment:
Please provide a proper description of this project for newbies, and mention the entry point of the code, for them to understand how to read this project.